backend/templates/users/signup.html

```diff
@@ -0,0 +1,37 @@
+{% extends 'base.html' %}
+
+{% block title %}Sign Up{% endblock %}
+
+{% block body %}
+{% if user_already_logged %}
```

Comment:
To check if there is a logged in user, I'd use this built-in Django helper: `user.is_authenticated`

https://docs.djangoproject.com/en/3.0/topics/auth/default/#users